docs/add-ons/velero.md

```diff
@@ -0,0 +1,69 @@
+# Velero
+
+Velero is a tool to backup and restore your Kubernetes cluster resources and persistent volumes. Velero lets you :
+
+Take backups of your cluster and restore in case of loss. Migrate cluster resources to other clusters. Replicate your production cluster to development and testing clusters.
+
+For complete project documentation, please visit the [Velero documentation site](https://velero.io/docs/v1.7/).
+
+## Usage
+
+[Velero](https://github.com/aws-samples/aws-eks-accelerator-for-terraform/tree/main/modules/kubernetes-addons/velero) can be deployed by enabling the add-on via the following.
+
+```hcl
+enable_velero= true
```

Comment:
Also show sample usage of `velero_helm_config` to demonstrate how customers can override default helm configuration if they wish to do so.